tests/unit/FileUploadPreview.spec.ts

```diff
@@ -271,24 +271,10 @@ describe('FileUploadPreview component', () => {
     inputFilesGet.mockReturnValue([oneMBFile])
     await fileInput.trigger('change')
     await waitForUpdate(3)
-    expect(wrapper.emitted('fileSelected').pop()[0]).toEqual(oneMBFile)
-
-    wrapper.destroy()
-  })
-
-  it('isFileValid event emitted when file is selected', async () => {
-    const wrapper = mount(FileUploadPreview, {
-      propsData: { maxSize: 10 * 1024 },
-      vuetify
-    })
 
-    const fileInput = wrapper.find('input[type="file"]')
-    setupFileInput(fileInput)
-    inputValue = oneMBFile.name
-    inputFilesGet.mockReturnValue([oneMBFile])
-    await fileInput.trigger('change')
-    await waitForUpdate(3)
-    expect(wrapper.emitted('fileValidity').pop()[0]).toEqual(true)
```

Comment:
Due to new check for valid PDF structure, this now emitted False.

Since this test was otherwise identical to the one above, I combined them.